src/gui/screen_printing_serial.cpp

```diff
@@ -75,50 +104,46 @@ void screen_printing_serial_init(screen_t *screen) {
         WINDOW_CLS_ICON, root,
         rect_ui16((240 - pt_ico.x) / 2, gui_defaults.msg_box_sz.y, pt_ico.x, pt_ico.y),
         &(pw->octo_icon));
-    //window_set_color_back(id, COLOR_GRAY);
     window_enable(id);
     window_set_icon_id(id, IDR_PNG_serial_printing);
     pw->octo_icon.win.f_enabled = 0;
     pw->octo_icon.win.f_disabled = 0;
 
-    //button tune
-    const rect_ui16_t rect_bt_tune = rect_ui16(8, 185, 64, 64);
-    id = window_create_ptr(
-        WINDOW_CLS_ICON, root,
-        rect_bt_tune,
-        &(pw->tune_button));
-    window_set_color_back(id, COLOR_GRAY);
-    window_enable(id);
-    window_set_icon_id(id, IDR_PNG_menu_icon_settings);
+    for (unsigned int col = 0; col < 3; col++) {
```

Comment:
here the `3` should be also replaced by a symbolic constant `number_of_buttons`, am I right?